### PR TITLE
Rename preferences to have 'group' prefix

### PR DESCRIPTION
**Pull request by jordanwade**
## DESCRIPTION
Simple clean-up PR that renames preferences to have a 'group' prefix. This is one of main refactors of preferences, but want to keep PRs targeted to avoid a giant PR to review. Note: preferences is not being used in Prod so no need to worry about breaking changes. Web-app-v2 will need to update the `useGroupPreference` to be `on GroupPreference` instead of `on Preference` and `on SubPreference`.

### 1. What does this PR do, or why is it needed?
Rename things based off feedback.

### 2. How do I test this PR?
Fire up web-app-v2 and make sure `/community` still shows preferences

## TODO:

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [x] No new warnings in tests, storybook, or apps
- [x] Assign relevant reviewers
- [ ] Upload GIF(s) of iOS, Android, Web
- [ ] Get one peer approval before merging

## REVIEW:

**Manual QA**

- [ ] QA branch on iOS and ensure it looks/behaves as expected
- [ ] QA branch on Android and ensure it looks/behaves as expected
- [ ] QA branch on Web and ensure it looks/behaves as expected

**Code Review**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Review by rkdavidson — COMMENTED** [review]
Ready to merge this pending this one gut check 👇

File: src/data/group-preferences/schema.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    id: ID!
+    title: String
+    summary: String
+    coverImage: ImageMedia
+    url: String
+  }
+
+  extend type Query {
+    allPreferences: [GroupPreference]
+    allSubPreferences: [GroupPreference]
```
[the remark this answers is not in the document]
Comment (jordanwade) [reply src/data/group-preferences/schema.js:14]: Yes, and this will evolve a bit more when I get to the feature setup for preferences. I figured there is no reason to have a separate type for SubPreferences since it is the same as Preferences minus a `summary`.
